src/OmniSharp/Settings/FromBodyApplicationModelConvention.cs

```diff
@@ -15,8 +15,8 @@ public void Apply(ApplicationModel application)
                     foreach (var parameter in action.Parameters)
                     {
                         if (parameter.BindingInfo?.BindingSource != null ||
-                            parameter.Attributes.OfType<IBindingSourceMetadata>().Any() ||
-                            ValueProviderResult.CanConvertFromString(parameter.ParameterInfo.ParameterType))
+                            parameter.Attributes.OfType<IBindingSourceMetadata>().Any())
+                            //|| ValueProviderResult.CanConvertFromString(parameter.ParameterInfo.ParameterType))
```

Comment:
There might be a better way to do this using application model instead of a filter.